dashboard/src/main/webapp/src/api/api.js

```diff
@@ -77,6 +77,6 @@ export function getTotalForUser(params,headers) {
 }
 
 // Obtain the configured coordinator server list
-export function getAllCoordinatorAddrees(params) {
-    return http.get('/coordinator/coordinatorList', params, {}, 1)
+export function getAllCoordinatorAddrees(params,headers) {
+    return http.get('/coordinator/coordinators', params, headers, 1)
```

Comment:
This is a little bit weird.

The RESTful style should be something like:
/plural noun
/plural noun/id

e.g.
/coordinators
/coordinators/1

You can refer to https://blog.restcase.com/5-basic-rest-api-design-guidelines/.
I don't know if it is convenient for you to refactor this.

Reply:
`/coordinator/coordinators` is indeed weird. Can you try to find a solution?